apps/site/pages/en/learn/userland-migrations/introduction.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Userland Migrations
+layout: learn
+authors: JakobJingleheimer, AugustinMauroy
+---
+
+![Node.js Userland Migrations](https://raw.githubusercontent.com/nodejs/userland-migrations/main/.github/assets/Userland-Migration-Tagline.png)
+
+# Userland Migrations
+
+Node.js provides migrations for "userland" (what you write vs node's own) source-code to facilitate adoption of new features and upgrading source-code affected by breaking changes. These are done in collaboration with [`codemod`](https://docs.codemod.com/introduction), who also work with other major projects like Next.js, React, and Tailwind. Node.js's migrations live in the [`nodejs/userland-migrations`](https://github.com/nodejs/userland-migrations) repository and are overseen by the `@nodejs/userland-migrations` team.
+
+<!--
+maintainer node: https://codemod.link/nodejs-official is pointing to legacy codemod registry
+it's will point to new new one once codemod change the redirection of the short link
+-->
```

Comment:
I don't like that we have a maintainer note, can we can an ETA on this change?

Reply:
cc @alexbit-codemod

Reply:
Agreed with Aviv. we don’t need to have such a note. 

The same shortlink will be updated to the new registry on August 8, 2025. 

(Besides link usage analytics, a key reason we use shortlinks is to avoid the maintenance overhead of updating links for users.)

cc @mohebifar